app/views/content/is-teaching-right-for-me/computing/_article.html.erb

```diff
@@ -0,0 +1,137 @@
+<div class="row inset">
+  <section class="col col-720">
+    <p>
+      Teaching computing is an important role to equip pupils with knowledge and skills that will help them succeed in a  world dependent on digital technology.</p>
```

Comment:
@caitlin-reidy Teaching computing is an **important role to equip** pupils with knowledge and skills.

The bit in bold felt slightly odd to me. Might be better to split this into 2 sentences? Teaching computing is an important role. You'll equip...

Reply:
@caitlin-reidy this intro sentence still feels a bit odd to me. Could we reword to make it a bit more plain English and inspirational? **skills that will help them succeed in a world dependent on digital technology.** what sort of skills? how will it help them?